core/domain/feedback_jobs_continuous_test.py

```diff
@@ -400,7 +400,8 @@ def test_thread_closed_job_running(self):
             })
 
             # Trigger close event.
-            threadlist = feedback_services.get_all_threads(exp_id, False)
+            threadlist = feedback_services.get_displayable_threads(exp_id,
```

Comment:
Suggest

    threadlist = feedback_services.get_displayable_threads(
        exp_id, False)

(and similarly below).